docs/server/settings.rst

```diff
@@ -501,7 +501,6 @@ Translation environment configuration settings.
   Current options:
 
   - Translate Toolkit (default) - translate.storage.statsdb.wordcount
-  - Pootle - pootle.core.utils.wordcount.wordcount
```

Comment:
I would remove `POOTLE_WORDCOUNT_FUNC` setting altogether if we are dropping this function.